package.json

```diff
@@ -13,7 +13,7 @@
     "envinfo": "dist/cli.js"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=6"
```

Comment:
let's experiment with how far back we can support without adding ridiculous numbers of polyfills.

I'll make a support table trying different targets and see what the bundle size is. 

I'd like to *try* to support node 4. It is after EOL but it was an LTS and still used a lot.

Reply:
Sure, but currently ci has no test for node 4.